docs/docker-images/buildkit-standalone.md

```diff
@@ -0,0 +1,73 @@
+This image contains `buildkit` with some Earthly-specific setup. This is what Earthly will start when using a local daemon. You can also start it up yourself and use it as a remote/shared buildkit daemon.
+
+*Note that versions of this container have only ever been tested with their corresponding version of `earthly`.* Mismatched versions are unspported.
+
+## Tags
+
+ * `prerelase`
+ * `v0.5.16`, `latest`
+ * `v0.5.15`
+ * `v0.5.14`
+
+## Quickstart
+
+Want to just get started? Here are a couple sample `docker run` commands that cover the most common use-cases:
+
+### Simple Usage (Use Locally)
+
+```bash
+docker run --privileged -t -v /tmp/earthly:/tmp/earthly:rw earthly/buildkitd:latest
+```
+
+Heres a quick breakdown:
+
+- `--privileged` is required. This is because `earthly` needs some privileged `buildkit` functionality.
+- `-t` tells Docker to emulate a TTY. This makes the `buildkit` log output colorized.
+- `-v /tmp/earthly:/tmp/earthly:rw` mounts the hosts `/tmp/earthly` into the containers `/tmp/earthly`. This is used as a temporary/working directory for `buildkitd` during builds.
+
+Assuming you are running this on your machine, you could use this `buildkitd` by setting `EARTHLY_BUILDKIT_HOST=docker-container://<container-name>`, or by specifying the appropriate values in `config.yml`.
+
+### Usage (Use As Remote)
+
+```bash
+docker run --privileged -t -v /tmp/earthly:/tmp/earthly:rw -e BUILDKIT_TCP_TRANSPORT_ENABLED=true -p 127.0.0.1:8372:8372 earthly/buildkitd:latest
```

Comment:
For remote usage, the user might want to `-p 8372:8372` to expose externally too, not just to `127.0.0.1`?